main.go

```diff
@@ -807,3 +902,21 @@ that will attach the file to your build as an artifact!`, logPth)
 		log.Warnf("Failed to export: BITRISE_XCODE_TEST_RESULT, error: %s", err)
 	}
 }
+
+func logDebugPretty(str string, v interface{}) {
+	if str != "" {
+		log.Debugf("%s: %+v\n", str, sprintFDebugPretty(v))
+		return
+	}
+
+	log.Debugf("%+v\n", sprintFDebugPretty(v))
+}
+
+func sprintFDebugPretty(v interface{}) string {
+	b, err := json.MarshalIndent(v, "", "  ")
+	if err != nil {
+		fmt.Printf("failed to parse %+v:\n error: %s", v, err)
```

Comment:
in the previously mentioned code, i use fmt.Sprint() if the json mashal would fail to make sure print something in every case